cli/Valet/PhpFpm.php

```diff
@@ -59,7 +59,7 @@ public function install(?string $version = null, bool $installExt = true): void
     {
         $version = $version ?: $this->getCurrentVersion();
         $version = $this->normalizePhpVersion($version);
-        if ($version === '') {
+        if ($version === '' && !($this->pm instanceof \Valet\PackageManagers\Pacman)) {
```

Comment:
This is a debatable change, instead of checking PackageManager here, can we do something to have a method on each PackageManager? So that in future if we introduce any other package manager than we can keep track of it.